openshift-infra/templates/tower_workflow_template_extra_vars.yml

```diff
@@ -1,7 +1,6 @@
 ---
 # Put any variables here that will be available to all members of this workflow
-lab_name: "{{ lab_name }}"
 lab_user: "{{ lab_user }}"
 student_id: "{{ student_id }}"
 openshift_deploy_version: "{{ openshift_deploy_version }}"
-domain_name: "{{ domain_name }}"
+openshift_cluster_public_url: "https{{':'}}//master-{{ student_id }}.{{ domain_name }}{{':'}}8443" 
```

Comment:
This is so the students can see the cluster URL when they execute the workflow. They can copy/paste it into a new tab. It doesn't get used anywhere or conflict with anything that I could tell.